README.md

```diff
@@ -169,8 +169,48 @@ bazel build --enable_runfiles build_pip_pkg
 TF_NEED_CUDA=1 bazel-bin/build_pip_pkg artifacts
 pip install artifacts/tensorflow_recommenders_addons_gpu-*.whl
 ```
+##### Apple Silicon Support
+**TFRA** Apple Silicon (M1) version requires:
+
+- macOS 12.0.0+
+- Python 3.9
+- tensorflow-macos 2.5.0
+- bazel 4.1.0+
+
+Before installing **TFRA** from source, you need to install tensorflow-macos from Apple. To install the natively supported version of tensorflow-macos, it's recommended to install the [Conda environment](https://github.com/conda-forge/miniforge). 
+
+After installing conda environment, run the following commands in the terminal.
+
+```sh
+# Create a virtual environment
+conda create -n $Your-Environment-Name Python=3.9
+
+# Activate your environment
+conda activate $Your-Environment-Name
+
+# Install TensorFlow macOS dependencies via miniforge
+conda install -c apple tensorflow-deps==2.5.0
+
+# Install base TensorFlow
+python -m pip install tensorflow-macos==2.5.0
+```
+
+There is a difference between the [tensorflow-macos installation instruction](https://developer.apple.com/metal/tensorflow-plugin/) and our instruction because this build requires Python 3.9 and tensorflow-macos 2.5.0.
+
+The building script has been tested on macOS Monterey, If you are using macOS Big Sur, you may need to customize the building script.
+
+```shell
+# Build arm64 wheel from source
+PY_VERSION="3.9" TF_VERSION="2.5.0" TF_NEED_CUDA="0" sh .github/workflows/make_wheel_macOS_arm64.sh
+
+# Install
+python -m pip install --no-deps ./artifacts/*.whl
+```
+
+The Apple silicon version TFRA doesn't support data type **float16**, the issue may be fixed in the future release.
```

Comment:
**NOTICE:**
- The Apple silicon version TFRA doesn't support data type **float16**, the issue may be fixed in the future release.

Reply:
Highlighted.